lib/controllers/skill-infrastructure-controller/index.js

```diff
@@ -140,12 +140,20 @@ module.exports = class SkillInfrastructureController {
      * @param {Function} callback (error)
      */
     updateSkillManifestWithDeployResult(rawDeployResult, callback) {
-        // 1.update local skill.json file: update the "uri" in all the existing "apis" for each region
-        R.keys(Manifest.getInstance().getApis()).forEach((domain) => {
+        const targetDomains = ResourcesConfig.getInstance().getTargetDomains(this.profile);
+        // for backward compatibility, defaulting to api from skill manifest if targetEndpoints is not defined
+        const domains = targetDomains.length ? targetDomains : Object.keys(Manifest.getInstance().getApis());
```

Comment:
duplicate logic. same logic in lambda deployer. not sure what is the best place to put